src/app/ui-helper/entity-subrecord/entity-subrecord.component.ts

```diff
@@ -140,4 +156,48 @@ export class EntitySubrecordComponent implements OnInit, OnChanges {
     col.selectValues = col.allSelectValues.filter(v => v.value.includes(input) || v.label.includes(input));
   }
 
+  // resets columnsToDisplay depending on current screensize
```

Comment:
if you type `/**` instead of `//` the IDE generates a code comment including parameters of the function, etc.
Using that to comment functions, I think we could even generate some documentation automatically if we ever need it.